dpdata/system.py

```diff
@@ -1231,6 +1233,11 @@ def from_qe_pw_scf(self, file_name) :
             = dpdata.qe.scf.get_frame(file_name)
         self.rot_lower_triangular()
 
+    @register_from_funcs.register_funcs('abacus/scf')
+    def from_qe_pw_scf(self, file_name) :
```

Comment:
This definition is overwritting line 1224. Please give the function a different name, for example `from_abacus_scf`